pkg/operator/ceph/object/bucket/rgw-handlers.go

```diff
@@ -47,10 +47,15 @@ func (p *Provisioner) createCephUser(username string) (accKey string, secKey str
 	return u.Keys[0].AccessKey, u.Keys[0].SecretKey, nil
 }
 
-func (p *Provisioner) genUserName(obcName, obcNamespace string) string {
-	// user name can be deterministically generated by obc name and namespace
-	// since there can't be 2 obcs in the same namespace with the same name, this will not collide
-	return "obc-" + obcNamespace + "-" + obcName
+func (p *Provisioner) genUserName(obcName, obcNamespace string) (string, error) {
```

Comment:
The `genUserName` is called by `GenerateUserID` func, But what is calling the `GenerateUserID`? 
https://github.com/rook/rook/blob/11e80b7047553c04eeeb5c11f1c6c76fda381d5e/pkg/operator/ceph/object/bucket/provisioner.go#L66

Reply:
It is called by https://github.com/kube-object-storage/lib-bucket-provisioner/blob/d1a8c34382f127670e363ae925ae6087e8b2c7bf/pkg/provisioner/controller.go#L324